server/rpc.go

```diff
@@ -50,6 +51,81 @@ type Broadcaster interface {
 	Sign([]byte) ([]byte, error)
 }
 
+type BroadcastSessionsManager struct {
+	broadcastSessions []*BroadcastSession
+	sessLock          *sync.Mutex
+}
+
+// func (bsm *BroadcastSessionsManager) selectFromList(node *core.LivepeerNode, pl core.PlaylistManager) *BroadcastSession {
+func (bsm *BroadcastSessionsManager) selectFromList() *BroadcastSession {
+	numOrchs := 3
```

Comment:
This hardcode will be removed once we determine how to generate this number for B's, keeping in mind available capacity on the network (which will be important especially in our current state, when the network is still growing).

Reply:
Technically `numOrchs` can be determined as a function of the segment length (2 secs now) and the request timeout (8 seconds now). This would give us at most 4 segments in-flight before earlier segments start to time out. Network capacity would be a concern if the number of available orchestrators is less than `numOrchs` as it would lead to list refreshes on every segment.

We can try to minimize this at some point with something like `numOrchs = min(numAvailableOrchs, defaultNumOrchs)`

Reply:
That sounds like a fine way to calculate a universal `numOrchs`, until we find a use case for allowing Orchs to set their own